connector/mqtt/vernemq/v2k-bridge/README.md

```diff
@@ -9,59 +14,82 @@ The V2K-bridge service implements a bridge between VerneMQ broker and Kafka brok
-V2K_MQTT_CLIENT_ID                 | MQTT client client id                                                             | hostname                              | string                         |
-V2K_MQTT_HOST                      | MQTT broker host                                                                  | vernemq-k8s                           | hostname/IP                    |
-V2K_MQTT_KEEPALIVE                 | MQTT client keepalilve                                                            | 60                                    | integer                        |
-V2K_MQTT_KEY_FILE                  | MQTT client key file                                                              | ${BASE_DIR}/app/verne/${HOSTNAME}.key | string                         |
-V2K_MQTT_USERNAME                  | MQTT client username                                                              | v2k-bridge                            | string                         |
-V2K_MQTT_PORT                      | MQTT broker port                                                                  | 8883                                  | integer                        |
-V2K_MQTT_SECURE                    | MQTT client secure                                                                | true                                  | true, false (case insensitive) |
-V2K_MQTT_SUBSCRIPTION_QOS          | MQTT client Quality of service                                                    | 1                                     | integer                        |
-V2K_MQTT_SUBSCRIPTION_TOPIC        | MQTT client topic to subscribe                                                    | $share/group/+/attrs                  | string                         |
-
-You can configure the Kafka Producer with variables from the Microservice SDK and librdkafka.
-For more details on these configurations,
-please read the [librdkafka official configuration guide](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md)
+Before running the V2K-bridge service within your environment, make sure you configure the
+environment variables to match your needs.
+
+### **App**
+
+Key                              | Purpose                                            | Default Value   | Valid Values     |
+-------------------------------- | -------------------------------------------------- | --------------- | ---------------- |
+V2K_APP_BASEDIR                  | Base directory where the project is located        | /opt/v2k_bridge | string           |
+V2K_APP_CONNECTION_RETRY_COUNT   | Number of retries when checking services health    | 3               | number           |
+V2K_APP_CONNECTION_RETRY_TIMEOUT | Seconds to wait between each health check          | 3               | number           |
+V2K_APP_EJBCA_ADDRESS            | Address of the EJBCA service                       | localhost:5583  | hostname/IP:port |
```

Comment:
In the future, maybe we can change this property because we are not communicating with EJBCA in fact, but a a service in front EJBCA (x509-identity-management).